staking/types/delegation.go

```diff
@@ -93,3 +93,20 @@ func (d *Delegation) DeleteEntry(epoch *big.Int) {
 		d.Entries = entries
 	}
 }
+
+// RemoveUnlockedUndelegations removes all fully unlocked undelegations and returns the total sum
+func (d *Delegation) RemoveUnlockedUndelegations(curEpoch *big.Int) *big.Int {
+	totalWithdraw := big.NewInt(0)
+	count := 0
+	for j := range d.Entries {
+		if curEpoch.Cmp(d.Entries[j].Epoch) > 14 { // need to wait at least 14 epochs to withdraw;
```

Comment:
maybe the 14 should be a named const? like in staking/effective

Reply:
oh wait, the Cmp gives back 1, 0, -1 right, so not the > 14

Reply:
Goot catch. Done.